crates/env/src/engine/off_chain/impls.rs

```diff
@@ -277,7 +277,7 @@ impl EnvInstance {
             beneficiary,
             transferred: all,
         };
-        panic!(scale::Encode::encode(&res));
+        panic!("{:?}", scale::Encode::encode(&res));
```

Comment:
Hmm, so I'm not sure.

I mean, even if we implement setting a tombstone, clearing storage, etc. we would still like to return the result of the contract eviction and it seems that `panic!` is the right way to go.

From my pov your suggestion would just be a semantic change of having something like `unimplemented!("we have not yet implemented everything, but anyway here's the result: …")` instead and changing that back to `panic!(…)` once we have implemented it would then be a breaking change.

Reply:
Well hopefully this discussion is not needed soon due to the works on the upcoming off-chain environment. :P But since contract termination is not really properly implemented in the current off-chain environment I would not dare to actually just mark it as unimplemented. But yeah let's keep it as it is for now.